src/SDL_gpu_shadercross.c

```diff
@@ -345,22 +436,27 @@ static void *SDL_ShaderCross_INTERNAL_CompileUsingDXC(
         &source,
         args,
         argCount,
-        NULL,
+        includeHandler,
         IID_IDxcResult,
         (void **)&dxcResult);
 
     SDL_free(entryPointUtf16);
+    if (includeDir != NULL) {
```

Comment:
Personally taste perhaps, but I'd use `includeDirUtf16` here.

```suggestion
    if (includeDirUtf16 != NULL) {
```